src/app/organizations/manage/people.component.ts

```diff
@@ -545,11 +548,14 @@ export class PeopleComponent implements OnInit {
         if (select) {
             this.selectAll(false);
         }
-        const selectCount = select && this.users.length > MaxCheckedCount
+
+        const filteredUsers = this.searchPipe.transform(this.users, this.searchText, 'name', 'email', 'id');
```

Comment:
How is the list itself actually getting filtered (before select all is clicked)? Isn't there a simpler or more direct way to get that list of items vs. re-filtering/re-searching the full list?

Reply:
The list is filtered in the template using
`<ng-container *ngIf="!loading && (isPaging() ? pagedUsers : users | search:searchText:'name':'email':'id') as searchedUsers">`.